ls1intum/Artemis
# `Exam mode`: Fix exam test run summary

### Checklist
#### General


#### Client

### Motivation and Context



Issue #5675 

### Description


When the exam is submitted during a test run, the user is forwarded to the exam summary page.
Furthermore, the "Review is open" alert is hidden for test runs.

### Steps for Testing

Prerequisites:
- 1 Instructor, 1 Student
- 1 Real Exam with multiple exercises of all kind

1. Log in to Artemis
2. Navigate to the Exam
3. Create a Test Run
4. Participate in the Test Run
5. Ensure you are forwarded to `/course-management/x/exams/x/test-runs/x/summary` when the exam is over/you submit the exam
6. Check that the "Review is open" alert no longer appears on the summary page
7. Ensure that normal exam conduction is unaffected
8. Repeat the steps with a _Test Exam_

### Review Progress





#### Code Review
#### Manual Tests

### Test Coverage


Added a test case for the routing redirect.

### Screenshots
